src/cosmoz-treenode.ts

```diff
@@ -1,6 +1,37 @@
 import { component, html } from '@pionjs/pion';
+import { Tree, Node } from '@neovici/cosmoz-tree/cosmoz-tree';
 
-export const computePathToRender = (path, hideFromRoot, showMaxNodes) => {
+export type ComputePathTextType = {
```

Comment:
We should not include the word `Type` in a type. We know its a type cause its TS.

Reply:
Since this uses some properties from Options we can reuse.  See https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys 
```suggestion
export interface ComputePathText extends Pick<Props, 'ownerTree'|'ellipsis'| ...> {
 // add here own props that aren't the same
 pathToRender?: string

}
```